internal/publish/model/exposure_model.go

```diff
@@ -92,37 +94,53 @@ func TruncateWindow(t time.Time, d time.Duration) time.Time {
 	return t.Truncate(d)
 }
 
+// TimeForIntervalNumber returns the time at which a specific interval starts.
+// This is done by turning the internal number into the corresponding unix timestamp,
+// multiplying by 600 seconds (10 minutes).
```

Comment:
Is this comment correct? I don't see the multiplication by 600s. It's kinda unclear to me what this is and why 10min.

Reply:
the comment was poorly written.